theano/gpuarray/opt.py

```diff
@@ -2069,7 +2085,7 @@ def provide_inputs(self, node, inputs):
                                          node.op.subsample,
                                          node.op.filter_dilation)
 
-            result[kshape] = theano.tensor.as_tensor_variable(node.op.kshp[2:])
+            result[kshape] = theano.tensor.as_tensor_variable(node.op.kshp[-2:])
```

Comment:
@affanv14 Do you know if this is tested anywhere? It doesn't seem to crash the `theano.gpuarray.tests.test_opt.Conv_opt_test`.